CHANGELOG.md

```diff
@@ -10,6 +10,14 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 Continuously importing events from a Zeek process with a low rate of
```

Comment:
Is this specific to the Zeek reader or to all text-based readers?

Reply:
Currently it's specific to zeek, although we should probably have a follow-up story to see if the same issue needs to be fixed for all our other readers as well.

Reply:
Okay.